client/controllers/incidentReports.coffee

```diff
@@ -67,7 +67,7 @@ Template.incidentReports.events
 
     incidentCount = if e.target.count then e.target.count.value.trim() else e.target.other.value.trim()
 
-    Meteor.call("addIncidentReport", templateInstance.data.userEvent._id, article, allLocations, e.target.incidentType.value, incidentCount, e.target.date.value, (error, result) ->
+    Meteor.call("addIncidentReport", templateInstance.data.userEvent._id, article, allLocations, e.target.incidentType.value, incidentCount, e.target.date.value, e.target.species.value, (error, result) ->
```

Comment:
I think it would be better to pass in this data as properties of a single object instead given the large number of arguments.